### ISSUE TITLE
I want to use sorting in the category page so I can change the order of the products to find these easily

### ISSUE THREAD

**Issue by rmakara** [DivanteLtd/shopware-pwa#6]
**Context**
After opening specific category I want to sort the products in the products list to make it easier for me to find an interesting product. 

**Dependencies**
* Blocked by #4 
* Blocked by #7 

**Acceptance criteria**
* I can see a list of available properties, that I can sort by for the current category.
* I can sort by [...] (to split by each sorting property to another stories?).
* When I use a sorting feature then products list gets updated (to extract to another story?).
* I can sort ascending (to extract to another story?).
* I can sort descending (to extract to another story?).
